packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -702,18 +702,32 @@ export const useTransactionHistory = (
     [updateCachedTransaction]
   )
 
+  const pause = useCallback(() => {
+    setFetching(false)
+  }, [setFetching])
+
+  const resume = useCallback(() => {
+    setFetching(true)
+    setPage(prevPage => prevPage + 1)
+  }, [setFetching, setPage])
+
   useEffect(() => {
     if (!runFetcher || !connector) {
       return
     }
-    connector.on('change', e => {
+    const handler = (e: ConnectorData<any>) => {
       // reset state on account change
       if (e.account) {
         setPage(1)
         setPauseCount(0)
         setFetching(true)
       }
-    })
+    }
```

Comment:
doesn't need to be in the useEffect??

Reply:
If you have it in the body of the component, it would be added on every render